rel8.cabal

```diff
@@ -114,6 +114,7 @@ library
     Rel8.Query.These
     Rel8.Query.Values
 
+    Rel8.Schema.Context.Lower
```

Comment:
I was sad to see this need to come back, but I'm happy to see it's going to be switfly removed :)